db/migrate/20170109115157_fix_lookup_key_auditable_type.rb

```diff
@@ -0,0 +1,15 @@
+class FixLookupKeyAuditableType < ActiveRecord::Migration
+  def up
+    Audit.reorder(nil).joins('JOIN lookup_keys ON lookup_keys.id = audits.auditable_id').
+         where(:auditable_type => 'LookupKey', :lookup_keys => {:type => 'VariableLookupKey'}).
+         update_all(:auditable_type => 'VariableLookupKey')
+
+    Audit.reorder(nil).joins('JOIN lookup_keys ON lookup_keys.id = audits.auditable_id').
+         where(:auditable_type => 'LookupKey', :lookup_keys => {:type => 'PuppetclassLookupKey'}).
+         update_all(:auditable_type => 'PuppetclassLookupKey')
```

Comment:
These can't be combined into one using `update_all('auditable_type = lookup_keys.type')` because of a bug in rails.